### ISSUE TITLE
Param name in GetClinicalIdentifier method in HKObject differs from older methods

### ISSUE THREAD

**Issue by mandel-macaque** [xamarin/xamarin-macios#4451]
_From @MichaelNorman on July 12, 2018 21:14_

In `HKObject`, we have a new method, `GetClinicalIdentifier`. For all the previous `GetFooIdentifier` methods, we've named the parameter along the lines of `hkFooTypeIdentifier`, to mirror Apple. However, we've settled on `identifier` for the new one. It seems like we should stay consistent, for aesthetic reasons. These show up among other suggestions in the Intellisense, as well as in tables in the online documentation.

Expected:
`hkFooTypeIdentifier` for parameter name.

Actual:
`identifier` is parameter name

_Copied from original issue: xamarin/maccore#883_

**Comment by mandel-macaque** [comment]
Thanks for the bug report. Unfortunately, this is the private repo for the tools related to the bindings. Will close the bug and will move it to the correct repo.

**Comment by VincentDondain** [comment]
@MichaelNorman Hi, I'm not sure which method you think is wrong, can you point me to something specific here https://github.com/xamarin/xamarin-macios/blob/xcode10/src/healthkit.cs#L838-L866? Or give me the native selector and I can search? We don't have `GetClinicalIdentifier` (at least it's not returned by my grep).

**Comment by MichaelNorman** [comment]
I erred on both the type and member, so you had no hope of finding it! The signature in the xml documentation (upon which I was relying) is `public HealthKit.HKClinicalType GetClinicalType (HealthKit.HKClinicalTypeIdentifier identifier);`.  I grepped on the latest sources this time. That member is, at least, present.

I apologize for the previous fire drill! I must have had a long day when I originally filed the issue.
